apps/zero-runtime-next-app/src/components/Slider/ZeroSlider.tsx

```diff
@@ -286,8 +301,8 @@ const SliderTrack = styled('span', {
           color: 'secondary',
         },
         style: {
-          backgroundColor: theme.vars ? theme.vars.palette.Slider.secondaryTrack : undefined,
-          borderColor: theme.vars ? theme.vars.palette.Slider.secondaryTrack : undefined,
+          backgroundColor: (theme.vars ?? theme).palette.Slider?.secondaryTrack,
```

Comment:
We should be able to not use this many conditional chaining if the types of the theme are correctly defined, no?

Reply:
Yes. That comes from the user. In this case, I specified vars as `{vars?: ThemeVars}` that's why the conditionals.